src/components/DebugMode.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="system-preferences">
+    <div class="checkbox">
```

Comment:
What does `<div class="checkbox">` do? I couldn't find any rules that use it (and it's unclear why it needs to be a child item).

I also couldn't spot what `<div class="system-preferences">` did.

But then, we end up with just a `<Checkbox>` by itself, with no useful extra logic, so we can get rid of this whole component…

Reply:
I wondered if it was possible to just use `<Checkbox>`, and then I didn't try it out. I tried it now and it looks good. Good thing you mentioned it :)